samples/azure/custom-partitioning-asb/ASB_7/Publisher/packages.config

```diff
@@ -4,8 +4,8 @@
   <package id="Microsoft.Data.OData" version="5.7.0" targetFramework="net452" />
   <package id="Microsoft.Data.Services.Client" version="5.7.0" targetFramework="net452" />
   <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net452" />
-  <package id="NServiceBus" version="6.0.3" targetFramework="net461" />
```

Comment:
Hmmm, not sure why this happened. We ( @boblangley  and I made sure to use the right "side").